pilot/pkg/serviceregistry/kube/controller/namespacecontroller_test.go

```diff
@@ -84,10 +85,10 @@ func updateNamespace(t *testing.T, client kubernetes.Interface, ns string, label
 	}
 }
 
-func expectConfigMap(t *testing.T, client kubernetes.Interface, ns string, data map[string]string) {
+func expectConfigMap(t *testing.T, client listerv1.ConfigMapLister, ns string, data map[string]string) {
 	t.Helper()
 	retry.UntilSuccessOrFail(t, func() error {
-		cm, err := client.CoreV1().ConfigMaps(ns).Get(context.TODO(), CACertNamespaceConfigMap, metav1.GetOptions{})
+		cm, err := client.ConfigMaps(ns).Get(CACertNamespaceConfigMap)
```

Comment:
@howardjohn thanks for identifying root cause and fix it.
nit: would you rename client to lister here?